vivisect/tests/testvivisect.py

```diff
@@ -7,31 +7,107 @@
 
 
 class VivisectTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
i can't say i'm a fan of this approach.  yes, i know i used it in my test cases, and loading everything up front is actually a pain, and seems to add complexity without much value.  that's my opinion, please enlighten me. :)  i'm sure there are reasons why this is a good approach, i just don't see it.  tbh, i'm not a fan of decorators without significant value, as they add complexity and break typical coding paradigms.

Reply:
I more use it so the analysis passes only happen once. If we were going to parallelize our tests, I'd say move these off into the init method of inside of an individual task. But for now, I'd like to avoid running them multiple times (since it's expensive and time consuming to do so).

Unless we do have plans to plumb through parallelizing unit tests, in which case my entire point for this is moot.